src/main/java/org/kitteh/irc/client/library/feature/twitch/event/WhisperEvent.java

```diff
@@ -0,0 +1,32 @@
+package org.kitteh.irc.client.library.feature.twitch.event;
+
+import org.kitteh.irc.client.library.Client;
+import org.kitteh.irc.client.library.element.ServerMessage;
+import org.kitteh.irc.client.library.element.User;
+import org.kitteh.irc.client.library.event.user.PrivateMessageEvent;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+
+/**
+ * Fires when a whisper is received via twitch.
+ */
+public class WhisperEvent extends PrivateMessageEvent implements TwitchSingleMessageEvent {
+    /**
+     * Creates the event.
+     *
+     * @param client client for which this is occurring
+     * @param originalMessages original messages
+     * @param sender who sent it
+     * @param target who received it
+     * @param message message sent
+     */
+    public WhisperEvent(@Nonnull Client client, @Nonnull List<ServerMessage> originalMessages, @Nonnull User sender, @Nonnull String target, @Nonnull String message) {
+        super(client, originalMessages, sender, target, message);
+    }
+
+    @Override
+    public void sendReply(@Nonnull String message) {
+        this.getClient().sendMessage("#jtv", "/w " + this.getActor().getNick() + " " + message);
```

Comment:
1. Is `#jtv` supposed to always be the target, or was this for testing? That feels weird, but this is Twitch we're dealing with.
2. Are you really supposed to send a `/w <target> <message>` to the channel itself? This feels even weirder. Comes out `PRIVMSG #jtv :/w mbaxter meow`

Reply:
Yes to both, this is _exactly_ how it's supposed to look. [Reference](https://discuss.dev.twitch.tv/t/whispers-on-irc/2459/3). Yeah, it is quite weird, but we just have to deal with it I guess.

Reply:
Twitch gonna twitch! Thanks.